README.md

```diff
@@ -1,22 +1,57 @@
+  <a href="logos/MetaBrainz Community/"> <img src="logos/All%20Logos/MetaBrainz%20Community.svg" alt="MetaBrainz Community" title="MetaBrainz Community" width="19%"></a>  
 </div>
 
-<div>
-  <img src="logos/All%20Logos/MusicBrainz.svg" alt="MusicBrainz" title="MusicBrainz" width="198">
-  <img src="logos/All%20Logos/ListenBrainz.svg" alt="ListenBrainz" title="ListenBrainz" width="198">
-  <img src="logos/All%20Logos/BookBrainz.svg" alt="BookBrainz" title="BookBrainz" width="198">
-  <img src="logos/All%20Logos/AccousticBrainz.svg" alt="AcousticBrainz" title="AcousticBrainz" width="198">
-  <img src="logos/All%20Logos/CritiqueBrainz.svg" alt="CritiqueBrainz" title="CritiqueBrainz" width="198">
-  <img src="logos/All%20Logos/MusicBrainz%20Picard.svg" alt="Picard" title="Picard" width="198">
-  <img src="logos/All%20Logos/CoverArtArchive.svg" alt="CoverArtArchive" title="CoverArtArchive" width="198">
-  <img src="logos/All%20Logos/MessyBrainz.svg" alt="MessyBrainz" title="MessyBrainz" width="198">
-  <img src="logos/All%20Logos/MetaBrainz%20Community.svg" alt="MetaBrainz Community" title="MetaBrainz Community" width="198">
-</div>
+# Guidelines
+
+## Sizes
+Use logo sizings consistently across projects. For instance, *Brainz logo mini* is used as the top left logo across all MB database project pages. When unsure, refer to prior projects and useage.</br>
+Although the icons should never be smaller than the minimum sizes\*, you may increase sizes as you wish. For instance, the usually small *Brainz icon* can be used in a very large format, if required.
+
+><img src="logos/MetaBrainz/SVG/MetaBrainz_logo_icon.svg" alt="MetaBrainz icon" height="24px"> / <img src="logos/MetaBrainz/SVG/MetaBrainz_logo_mini.svg" alt="MetaBrainz logo mini" height="23px"></br>
+>*Brainz icon / Brainz logo mini*</br>
+>No minimum
+
+><img src="icons/metabrainz-icon-detail.svg" alt="MetaBrainz icon detail" height="34px"> / <img src="logos/MetaBrainz/SVG/MetaBrainz_logo_small.svg" alt="MetaBrainz logo small" height="33px"></br>
+>*Brainz icon detail / Brainz logo small*</br>
+>Minimum height: 33px/12mm</br>
+>Under this size details can get unclear and distracting, particularly for visually impaired users.
+
+><img src="logos/MetaBrainz/SVG/MetaBrainz_logo_no_text.svg" alt="MetaBrainz logo no text" height="58px"> / <img src="logos/MetaBrainz/SVG/MetaBrainz_logo.svg" alt="MetaBrainz logo" height="55px"></br>
+>*Brainz logo no text / Brainz logo*</br>
+>Minimum height: 55px/20mm</br>
+>Ideally has generous sizing and space to breathe. If the page/app/context is very busy, consider using a simpler icon instead.
+
+\*a logo may display fine at a smaller size, but other project logos have more detail, and won’t. Smaller sizes makes consistent logo display across projects difficult and is to be avoided. However there may be fringe cases it is appropriate.
```

Comment:
```suggestion
\*a specific logo may display fine at a smaller size, but other project logos have more detail, and won’t. Smaller sizes make consistent logo display across projects difficult and are to be avoided. However there may be fringe cases where using smaller sizes is appropriate.
```

Reply:
Thanks Reo! I had pretty much *exactly* written what you had, but shortened it! Very happy to have a second eye on it all, thank you.

Question: If I hit 'commit suggestion' on all of these is it making the pull request complicated again/do I have to squash stuff? Or am I overthinking it and can just 'commit suggestion' and not worry about it.

Reply:
“Commit suggestion” will just add a commit to your pull request, so it is usually safe to click on it. If you want to keep your pull request tidy, you can always reorganize the commits after pulling the branch from GitHub to your local working copy.